manual/cli.rst

```diff
@@ -1,4 +1,100 @@
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| --bw       | --black-and-white    | Draw maps in black and white                                                                                                  |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+
+
+Options valid only for generate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| Short     | Long                       | Description                                                                                          |
++===========+============================+======================================================================================================+
+| -r FILE   | --rivers=FILE              | produce a map of rivers, after the option it expects the name of the file where to generate the map  |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| --gs      | --grayscale-heightmap      | produce a grayscale heightmap                                                                        |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --ocean_level=N            |  elevation cut off for sea level [default = 1.0]                                                     |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --temps #/#/#/#/#/#        |  Provide alternate ranges for temperatures. [default = .126/.235/.406/.561/.634/.876]                |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --humidity #/#/#/#/#/#/#   | Provide alternate ranges for humidities. [default = .059/.222/.493/.764/.927/.986/.998]              |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| -gv N     | --gamma-value N            | N = Gamma value for temperature/precipitation gamma correction curve. [default = 1.25]               |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| -go N     | --gamma-offset N           | N = Adjustment value for temperature/precipitation gamma correction curve. [default = .2]            |
```

Comment:
Sorry, things came up that dragged me away for a bit.

I will try and put something together. Unfortunately the whole process is a little complicated. First I need to explain that the different biomes are chosen by the intersection of humidity and temperature values. Then I have to explain that both temperature and humidity and unitless values (because that's not mentioned anywhere). With those explained I can then explain how the numbers are used to map the unitless values into those ranges.

Unfortunately RST doesn't seem to support coloring the background of cells in any convenient way, which means the current biome chart I created will probably have to be handled as an image. Additionally I need to create an image of the Holdridge life zones that corresponds to our particular implementation.